float64[] pressures
duration time_from_start